### PR TITLE
DMAPI 3 patch. Fix Discord mention parsing

### PR DESCRIPTION
**Pull request by Cyberboss**
On the tin

### CODE REVIEW

File: code/modules/tgs/v3210/commands.dm [hunk 1]
```diff
@@ -42,5 +42,11 @@
 	var/datum/tgs_chat_command/stc = new command_type
 	var/datum/tgs_chat_user/user = new
 	user.friendly_name = sender
+
+	// Discord hack, fix the mention if it's only numbers (fuck you IRC trolls)
+	var/regex/discord_id_regex = regex(@"[0-9]+")
```
Comment (bobbahbrown) [inline code/modules/tgs/v3210/commands.dm:47]:
```suggestion
	var/regex/discord_id_regex = regex(@"^[0-9]+$")
```

**Review by SpaceManiac — CHANGES_REQUESTED** [review]
please confirm/deny what bobbah pointed out

**Comment by Cyberboss** [comment]
His is better, but I need to patchit upstream

**Comment by SpaceManiac** [comment]
You can reopen this when you fix it

**Pull request closed without merging**